ush/wave_tar.sh

```diff
@@ -124,21 +130,15 @@ source "${USHgfs}/preamble.sh"
         exit 3
       fi
       
-      if [ -f "$ID.$cycle.${type}_tar" ]
+      if [[ -f "${ID}.${cycle}.${type}_tar" ]]
       then
         tardone='yes'
       fi
-    else
```

Comment:
I see that a lot of the wave post point jobs are failing in CI.  I didn't see the errors but looking again at this code, it looks like the logic here was changed.  Perhaps that's the issue?